content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -1251,6 +1251,7 @@ const deleteUser = await prisma.user.delete({
 #### Remarks
 
 - `createMany` is not supported by SQLite.
+- The `skipDuplicates` option is not supported by MongoDB.
```

Comment:
You might want to add that in line 1243 as well in some form - technically it is already there, but seems not obvious enough.